index.html

```diff
@@ -4,10 +4,48 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Personal Portfolio</title>
+    <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
-<body>
-    
+<body class="flex_column">
+    <header class="flex">
+        <nav class="logo flex_column">
+            <a href=""><p>My logo</h2></a>
+        </nav>
+        <nav class="flex">
+            <a href="#" class="burgeur flex"><img src="assets/Icon - Menu.png" alt="menu"></a>
+        </nav>
+    </header>
+
+    <main>
+        <section id="Home">
+            <div class="headline flex_column">
+                <div class="description">
+                    <h1>Hey There <br> I'm Ben.</h1>
+                    <h2>I'm a Software Developer</h2>
+                    <p>I can help you build a product , feature or website.
+                         Look through some of my work and experience! 
+                         If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                </div>
+                <ul class="flex">
+                    <li><a href="https://github.com/BenM4k"><img src="assets/github.svg" alt="github"></a></li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/b%C3%A9n%C3%A9dict-makomo-885856248/">
+                            <img src="assets/Linkedin icon.svg" alt="linkedin">
+                        </a>
+                    </li>
+                    <li><a href="https://angel.co"><img src="assets/angel list.svg" alt="angel"></a></li>
+                    <li><a href="https://twitter.com"><img src="assets/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="https://medium.com"><img src="assets/medium.svg" alt="medium"></a></li>
+                </ul>
+                <img src="assets/Group.png" alt="bg" class="bg_img">
```

Comment:
- [OPTIONAL] As other code reviewers mentioned, this is the background image and should be added as a `background-image` property in the CSS file. It is still not responsive to the page and it makes the mobile page larger than the mobile screen, making it possible for the user to scroll horizontally creating a bad user experience. It would be nice to see the change of this image position and responsiveness in the CSS file as a `background-image`.
  - Your design:
  - 
<img width="432" alt="image" src="https://user-images.githubusercontent.com/61919617/227182382-b10cf4db-97d4-4773-b87b-4f707089f5aa.png">

Reply:
I'm having difficulties adding transform to a background-image property